src/index.js

```diff
@@ -100,14 +108,16 @@ module.exports = postcss.plugin('css-declaration-sorter', function (options) {
 
     options = options || {};
 
+    const position = options.position || 'top';
```

Comment:
I made this functionality with flag. And if you don't like new behaviour by default, we can change `top` to `bottom` but new ordering will be available too.